runtime/cmd/decode-state-values/main.go

```diff
@@ -258,111 +259,137 @@ func load() {
 
 	bar := progressbar.Default(int64(len(storage)))
 
+	var slabNotFoundErrCount int
+
 	for storageKey, data := range storage { //nolint:maprangecheck
 		_ = bar.Add(1)
 
-		var isStoragePath bool
-
 		// Check the key is a non-root slab or a storage path
 		key := storageKey[2]
 
 		var address atree.Address
 		copy(address[:], storageKey[0])
 
-		// If the key is for a slab (format '$' + storage index),
-		// then attempt to decode the slab
+		err := loadStorageKey(key, address, data, inter, slabStorage)
+		var slabNotFoundErr *atree.SlabNotFoundError
```

Comment:
I'm not sure I understand when `loadStorageKey` would return `SlabNotFoundError`.